README.md

```diff
@@ -1 +1,4 @@
 # Hej, världen!
+This is octo-cat trolling.
```

Comment:
I think you should add an empty line after this line. And also you change the name into `swapper3d`.

- [ ] Add empty line
- [ ] Change the name.